pkg/agent/controller/networkpolicy/allocator_test.go

```diff
@@ -222,15 +223,16 @@ func TestWorker(t *testing.T) {
 				}
 				return false, nil
 			}
-			if err := wait.PollImmediate(time.Millisecond, time.Millisecond*10, conditionFunc); err != nil {
+
+			if err := wait.PollImmediate(time.Millisecond, time.Second, conditionFunc); err != nil {
 				t.Fatalf("Expect the rule with id %v to be deleted from async rule cache", tt.expectedID)
```

Comment:
should we validate the time it takes for the rule to be deleted? With the current code the test will pass even if the deletion takes 1s. That's also why I suggested using longer time intervals.

Reply:
Used epsilonf to validate the time.